validator/slashing-protection/local/standard-protection-format/export.go

```diff
@@ -0,0 +1,88 @@
+package interchangeformat
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/prysmaticlabs/prysm/validator/db"
+)
+
+// ExportStandardProtectionJSON extracts all slashing protection data from a validator database
+// and packages it into an EIP-3076 compliant, standard
+func ExportStandardProtectionJSON(ctx context.Context, validatorDB db.Database) (*EIPSlashingProtectionFormat, error) {
+	interchangeJSON := &EIPSlashingProtectionFormat{}
+	genesisValidatorsRoot, err := validatorDB.GenesisValidatorsRoot(ctx)
+	if err != nil {
+		return nil, err
+	}
+	genesisRootHex, err := rootToHexString(genesisValidatorsRoot)
+	if err != nil {
+		return nil, err
+	}
+	interchangeJSON.Metadata.GenesisValidatorsRoot = genesisRootHex
+	interchangeJSON.Metadata.InterchangeFormatVersion = INTERCHANGE_FORMAT_VERSION
+
+	// Extract the existing public keys in our database.
+	proposedPublicKeys, err := validatorDB.ProposedPublicKeys(ctx)
+	if err != nil {
+		return nil, err
+	}
+	dataByPubKey := make(map[[48]byte]*ProtectionData)
```

Comment:
```suggestion
	dataByPubKey := make(map[[48]byte]*ProtectionData, len(proposedPublicKeys))
```

Reply:
It actually also depends on attested public keys, so will do in a separate PR